exp/zapslog/handler_test.go

```diff
@@ -189,3 +193,42 @@ func TestAttrKinds(t *testing.T) {
 		},
 		entry.ContextMap())
 }
+
+func TestSlogtest(t *testing.T) {
+	t.Skip() // TODO: skip for now until test failures are fixed (#1334)
```

Comment:
I would prefer to keep this test off master until we actually have it passing.
`t.Skip`s don't really get follow ups that often—I'd prefer we track the pending work in issues as we're already doing that.

Would you mind deleting this change from this PR?